topics/contributing/tutorials/slides-with-video/tutorial.md

```diff
@@ -75,30 +75,41 @@ The captioning process is completely automated, but it means that for very long
 
 Every sentence must end with punctuation like `.` or `?` or even `!` if you're feeling excited. We merge all lines of the presenter notes before having them spoken by Polly. If you forget the punctuation it will be one long run-on sentence.
 
-### Abbreviations and "Weird" Names
+### Abbreviations
 
-In the captions you will want to spell out abbreviations
+These are generally fine as-is. Make sure they are all caps though.
 
-> > **Good**
-> > This role deploys C V M F S.
-> {: .code-in}
->
-> > **Bad**
-> > This role deploys CVMFS.
-> {: .code-out}
-{: .code-2col}
+> **Good**
+> This role deploys CVMFS.
+{: .code-in}
 
-For many terms we read them differently than they're written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
+### "Weird" Names
 
-> > **Good**
-> > Copy copies a file from src on localhost...
-> {: .code-in}
->
-> > **Bad**
-> > Copy copies a file from source on localhost...
-> {: .code-out}
-{: .code-2col}
+In the captions you will want to teach the GTN how to pronounce these words by editing `bin/ari-map.yml` to provide your definition.
+
+E.g.
+
+Word       | Pronunciation
+---------- | ---
+SQLAlchemy | SQL alchemy
+FastQC     | fast QC
+nginx      | engine X
+gxadmin    | GX admin
+/etc       | / E T C
+
+The same applies to the many terms we read differently from how they are written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
+
+So we write the definition in the [`bin/ari-map.yml`](https://github.com/galaxyproject/training-material/blob/master/bin/ari-map.yml) file.
+
+### Other Considerations
+
+(*Written 2020-12-16, things may have changed since.*)
+
+Be sure to check the pronunciation of the slides. There are known issues with [Homographs](https://en.wikipedia.org/wiki/Homograph), words spelt the same but pronounced differently often depending on context. Consider "read" for a classic example, or ["analyses"](https://en.wiktionary.org/wiki/analyses#English) for one that comes up often in the GTN. "She analyses data" and "Multiple analyses" are pronounced quite differently based on their usage in sentences. See the [wiktionary](https://en.wiktionary.org/wiki/analyses#English) page for more information, or the [list of English homographs](https://en.wikipedia.org/wiki/List_of_English_homographs) you might want to be aware of.
```

Comment:
```suggestion
Be sure to check the pronunciation of the slides. There are known issues with [heteronyms](https://en.wikipedia.org/wiki/Heteronym_(linguistics)), words spelt the same but having different pronunciation and meaning. Consider "read" for a classic example, or ["analyses"](https://en.wiktionary.org/wiki/analyses#English) for one that comes up often in the GTN. "She analyses data" and "Multiple analyses" are pronounced quite differently based on their usage in sentences. See the [wiktionary](https://en.wiktionary.org/wiki/analyses#English) page for more information, or the [list of English heteronyms](https://en.wiktionary.org/wiki/Category:English_heteronyms) you might want to be aware of.
```

Reply:
Ah i picked the wrong one from their venn diagram. Cheers.